src/ADAL.PCL.WinRT/PlatformInformation.cs

```diff
@@ -81,6 +81,11 @@ public override string GetOperatingSystem()
 
         public override string GetDeviceModel()
         {
+            return null;
+        }
+
+        public override string GetDeviceId()
```

Comment:
Returning deviceInformation.id.

---
In reply to: [74538089](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r74538089) [](ancestors = 74538089)